test/base.py

```diff
@@ -45,33 +48,36 @@ def flatten(l, ltypes=(list, tuple)):
         i += 1
     return ltype(l)
 
+
 class ColumnName(str):
     pass
 
+
 def dbparam(x):
     if x is None:
         return 'NULL'
     elif isinstance(x, ColumnName):
         return "`" + x + "`"
-    elif isinstance(x, basestring):
+    elif isinstance(x, bytes):
```

Comment:
I don't think it catches unicode, both bytes and str are the same thing in py27.